Sources/Specs/dictionaryManagerSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Dictionary
+import Swiftest
+
+class DictionaryManagerSpec: Swiftest.Spec {
+	let spec = describe("dictionary logic") {
+		var dictionaryManager: DictionaryManager!
+		before() {
+			dictionaryManager = DictionaryManager()
+		}
+
```

Comment:
what's going on here? why are there 2 groupings of variables and before hooks for a single test? You should just move the variable assignments into the single test.